### PR TITLE
Add a session secret and the missing RBACs to the gcsweb-private deployment

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: clusters/app.ci/prow/03_deployment/gcsweb-private.yaml [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
-apiVersion: v1
-items:
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- description: The session secret for the component.
+  name: SESSION_SECRET
+  generate: expression
+  from: "[a-zA-Z0-9]{43}"
+- description: namespace
```
Comment (alvaroaleman) [inline clusters/app.ci/prow/03_deployment/gcsweb-private.yaml:8]: You need to use the namespace variable

Comment (droslean) [reply clusters/app.ci/prow/03_deployment/gcsweb-private.yaml:8]: adressed

**Pull request merged**